src/virtualenv/info.py

```diff
@@ -0,0 +1,26 @@
+from __future__ import absolute_import, unicode_literals
+
+import sys
+
+from appdirs import user_config_dir, user_data_dir
+
+from virtualenv.util.path import Path
+
+IS_PYPY = hasattr(sys, "pypy_version_info")
+PY3 = sys.version_info[0] == 3
```

Comment:
Guido spoke up that we'll continue having 3.10 instead of 4.x, and until there's a clear signal from the steering council that 4.x will not be a breaking change I consider this to be the correct way of checking for Python 3 compatibility 👍

Reply:
Even still, this code is likely to live much longer than normal code -- I'd strongly suggest using `>=` here for future proofing

It is very unlikely that python4 will be breaking in the same manner as python3 was -- guido has also spoken in the past that when python4 comes it will be "just another python3"

The main reason for preferring python3.10 is because people write incorrect code like `== 3` which prevents the ecosystem from migrating